clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1176,10 +1176,11 @@ public static String toHtml() {
         for (ApiKeys key : ApiKeys.values()) {
             // Key
             b.append("<h5>");
+            b.append("<a name=\"The_Messages_" + key.name + "\">");
```

Comment:
Thanks @ewencp for your feedback.

According to [HTML5.1 spec](https://www.w3.org/TR/html/browsers.html#navigating-to-a-fragment-identifier) it seems both `id` and `name` attributes can be used to identify a fragment in the DOM. The existence of an element with the given `id` attribute is checked first and then the existence of an `<a>` element with the given `name` attribute:

> 5. If there is an element in the DOM that has an ID exactly equal to decoded fragid, then the first such element in tree order is the indicated part of the document; stop the algorithm here.
> 6. No decoded fragid: If there is an a element in the DOM that has a name attribute whose value is exactly equal to fragid (not decoded fragid), then the first such element in tree order is the indicated part of the document; stop the algorithm here.

I think a `name` attribute might be a better fit here since we are dealing with `<a>` elements, but the fragment ids we are using should be unique and `id` attributes should work too.

I'm fine both ways. What's your opinion? Thanks.